test/jdk/java/awt/print/PrinterJob/PageDialogMarginTest.java

```diff
@@ -33,41 +33,42 @@
 import java.awt.print.PrinterJob;
 import javax.print.attribute.HashPrintRequestAttributeSet;
 import javax.print.attribute.PrintRequestAttributeSet;
+import javax.print.attribute.standard.DialogTypeSelection;
 import javax.print.attribute.standard.MediaPrintableArea;
 import javax.swing.JOptionPane;
 import javax.swing.SwingUtilities;
+import java.text.MessageFormat;
```

Comment:
I wonder where do DialogTypeSelection and MessageFormat come from?

The imports should be sorted by default, usually your IDE handles it perfectly on its own.